packages/dapp/.eslintrc

```diff
@@ -1,18 +1,9 @@
 {
   "root": true,
   "parser": "@typescript-eslint/parser",
-  "plugins": [
-    "@typescript-eslint",
-    "prettier"
-  ],
-  "extends": [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
-    "prettier"
-  ],
-  "ignorePatterns": [
-    "contracts/"
-  ],
+  "plugins": ["@typescript-eslint", "prettier"],
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/recommended", "prettier"],
+  "ignorePatterns": ["contracts/"],
```

Comment:
Maybe this should also be renamed to `eslint.config.json`

Reply:
> I noticed that we have the following formatter config file names:
> 
> ```
> 	.eslintrc.json
> 	.prettierrc.json
> 	packages/dapp/.eslintrc.json
> ```
> 
> The last one, in `dapp/` seems redundant? Can we remove it?



> I noticed that we have the following formatter config file names:
> 
> ```
> 	.eslintrc.json
> 	.prettierrc.json
> 	packages/dapp/.eslintrc.json
> ```
> 
> The last one, in `dapp/` seems redundant? Can we remove it?

the one on dapp looks redundant

Reply:
I renamed this to `.eslintrc.json`